### ISSUE TITLE
Prioritize OrderProcessors

### ISSUE THREAD

**Issue by gorkalaucirica** [Sylius/Sylius#6988]
The order processor have no priority at all, causing some issues:
- Cannot add a custom order processor in between existing processors
- For custom checkouts some values end up been wrong in the database, such as payments.

What happened to us is that we faced some issues due to this priority:
```
Sylius\Component\Core\OrderProcessing\OrderExchangeRateProcessor
Sylius\Component\Core\OrderProcessing\OrderShipmentProcessor
Sylius\Component\Core\OrderProcessing\OrderPaymentProcessor
Sylius\Component\Core\OrderProcessing\OrderAdjustmentsClearer
Sylius\Component\Core\OrderProcessing\ShippingChargesProcessor
Sylius\Component\Core\OrderProcessing\OrderPricesRecalculator
Sylius\Component\Core\OrderProcessing\OrderPromotionProcessor
Sylius\Component\Core\OrderProcessing\OrderTaxesProcessor
```
We have a custom order processor that selects a shipping method automagically. This processor is prioritized to be the first one, but `ShippingChargesProcessor` is processed after `OrderPaymentProcessor`. Therefore the payment amount and order amount are inconsistent in db.

The same happens with promotions, taxes or adjustments.

In conclusion, `OrderPaymentProcessor` should be the last and we should leave space between processors to allow custom order processors.

Can open a PR if you consider this should be changed.

**Comment by pjedrzejewski** [comment]
Absolutely, we should have priorities allowing enough space for custom processors. 👍  Thanks for the suggestions. :)

**Comment by gorkalaucirica** [comment]
Done in #6994
